### DISCUSSION TITLE
hope custom class names to support colons

### DISCUSSION THREAD

**Discussion by missannil** [tailwindlabs/tailwindcss#11169] [Ideas]
A component accepts incoming classes ，like this

<button  classes="{{ external_class}}"   /> 

below is In the ‘.js’ section of the page that uses this component

```js
defineComponent({
    prop:{
        external_class:{
           type: String,
           value: 'w-16' // No prompt currently
       }
    }
})
```
configuration files
```json
    //vscode setting.json
    {
    //...
    "tailwindCSS.classAttributes":['class','value',]
    }
```
hope the "value:" field has hint 。

thanks！

**Comment by wongjn** [comment]
You could try configuring the [`tailwindCSS.experimental.classRegex` setting](https://github.com/tailwindlabs/tailwindcss/issues/7553#issuecomment-735915659).
